src/main/groovy/ca/stellardrift/gitpatcher/GitPatcherExtension.java

```diff
@@ -64,4 +65,12 @@ public interface GitPatcherExtension {
      * @since 1.1.0
      */
     Property<String> getCommitterEmailOverride();
+
+    /**
+     * The extra command line arguments which will be passed to Git am command
+     * when applying patches.
+     *
+     * @return the extra arguments which will be passed to Git am command
+     */
+    ListProperty<String> getAMExtraArguments();
```

Comment:
I don't like the use of the `AM` abbreviation here -- the fully correct option would be `getApplyMailboxExtraArguments()`, but that feels overly verbose so how about `getApplyExtraArguments()` while specifying that the arguments are applied to the apply mailbox operation in the JD?

Reply:
It would be great. I will do so.